doc/developer_guide/APIs.rst

```diff
@@ -1041,9 +1041,19 @@ layer routine. A lot of this work is currently performed in the
 https://github.com/stfc/PSyclone/issues/1258) much of this will be
 removed.
 
-To date, the following LFRic BuiltIns have had
-``lower_to_language_level()`` methods implemented: ``LFRicXPlusYKern``,
-``LFRicIncXPlusYKern``, and ``LFRicAPlusXKern``.
+To date, all the LFRic BuiltIns have had ``lower_to_language_level()``
+methods implemented except for the following:
+
+* ``LFRicXInnerproductYKern``,
+* ``LFRicXInnerproductXKern``,
+* ``LFRicSumXKern``,
+* ``LFRicIntXKern``,
+* ``LFRicRealXKern``.
+
+The sum and inner product BuiltIns require implementing PSyIR representation
```

Comment:
The 'R' in 'PSyIR' stands for 'Representation' so you don't need "representation". Also, please could you say what needs to be implemented or, alternatively, simplify and just say that the PSyIR needs to be extended to handle reductions.